lib/parse/push.rb

```diff
@@ -17,32 +16,35 @@ class Push
     def initialize(data, channel = '', client = nil)
       @data = data
 
+      # NOTE: if no channel is specified, by setting "where" to an empty clause
+      #   a push is sent to all clients.
       if !channel || channel.empty?
-        # If no channel is specified, by setting "where" to an empty clause, a push is sent to all clients.
         @where = {}
       else
-        @channel = channel
+        @channels = [channel]
```

Comment:
May be it's better to do this like `@channels = *channel` in order to be able to pass array of channels on initialization? Also it looks strange for me that we are using empty string as a default value for channel param `def initialize(data, channel = '', client = nil)`. Why not to use `nil` instead?